ddocs/medic/_attachments/appcache-upgrade.html

```diff
@@ -0,0 +1,6 @@
+<html>
+  <head>
+    <script type="text/javascript" src="/appcache-upgrade.js"></script>
```

Comment:
Why not just inline the JS here? I don't think there's any point making a separate request, is there?

Reply:
The CSP would mean we'd have to maintain a whitelisted hash of the inline script. Not sure which is cleaner.

Reply:
Of course... I always forget about the CSP.

I wonder if the refresh meta tag would work, eg:

```html
<meta http-equiv="refresh" content="0; url=/" />
```

Reply:
Ya. Seems to work fine. Tested on mobile + desktop.